account_mass_reconcile/models/mass_reconcile_history.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+# © 2012-2016 Camptocamp SA (Guewen Baconnier, Damien Crier, Matthieu Dietrich)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import models, api, fields, _
+
+
+class MassReconcileHistory(models.Model):
+    """ Store an history of the runs per profile
+    Each history stores the list of reconciliations done"""
+
+    _name = 'mass.reconcile.history'
+    _rec_name = 'mass_reconcile_id'
+    _order = 'date DESC'
+
+    @api.one
+    @api.depends('reconcile_ids')
+    def _get_reconcile_line_ids(self):
+        move_line_ids = []
+        for reconcile in self.reconcile_ids:
+            move_lines = reconcile.mapped('reconciled_line_ids')
+            move_line_ids.extend(move_lines.ids)
+        self.reconcile_line_ids = move_line_ids
+
+    mass_reconcile_id = fields.Many2one(
+        'account.mass.reconcile',
+        string='Reconcile Profile',
+        readonly=True
+    )
+    date = fields.Datetime(string='Run date', readonly=True, required=True)
+    reconcile_ids = fields.Many2many(
+        comodel_name='account.full.reconcile',
+        relation='account_full_reconcile_history_rel',
```

Comment:
as it was specified in the previous version, specifying the relation will help for migrations